apps/infrastructure/src/ecs.tf

```
@@ -32,6 +32,8 @@ data "template_file" "cms_app" {
     workflow_id          = "deploy_website.yaml"
     target_branch        = "main"
     github_pat           = aws_ssm_parameter.cms_github_pat.arn
+    preview_token        = aws_ssm_parameter.preview_token.arn
+    preview_url          = "https://preview.${keys(var.dns_domain_name)[0]}"
```

```suggestion
    preview_url          = "https://preview.${keys(var.dns_domain_name)[0]}/preview"
```
The preview won't be served from the base URL "https://preview.b2bportals.pagopa.it", but from "https://preview.b2bportals.pagopa.it/preview" as suggested.